components/dashboard/src/app/AppRoutes.tsx

```diff
@@ -99,10 +95,6 @@ export const AppRoutes = () => {
         return <OAuthClientApproval />;
     }
 
-    if (isWhatsNewShown) {
-        return <WhatsNew onClose={() => setWhatsNewShown(false)} />;
```

Comment:
I'm not sure as well what it is and why it is alright to drop.

Reply:
Same as https://github.com/gitpod-io/gitpod/pull/19142#discussion_r1415625044